x/leverage/keeper/borrows.go

```diff
@@ -108,6 +108,38 @@ func (k Keeper) CalculateBorrowLimit(ctx sdk.Context, collateral sdk.Coins) (sdk
 	return limit, nil
 }
 
+// CalculateLiquidationThreshold uses the price oracle to determine the liquidation threshold
+// (in USD) provided by collateral sdk.Coins, using each token's uToken exchange rate and
+// liquidation threshold. An error is returned if any input coins are not uTokens or if value
+// calculation fails.
+func (k Keeper) CalculateLiquidationThreshold(ctx sdk.Context, collateral sdk.Coins) (sdk.Dec, error) {
```

Comment:
How about `CalculateBorrowerLiquidationThreshold`?

Reply:
You don't think `Value` should be in there? Because threshold, the param, implies a %, whereas this returns a USD value.

Reply:
True - also if there were a way to incorporate `Limit` into the name it would also match `BorrowLimit`, but `LiquidationLimit` and `LiquidationValue` both sound like they apply to the liquidation transaction itself.